.github/workflows/test.yml

```diff
@@ -41,5 +39,40 @@ jobs:
     - name: Install dependencies
       run: sudo apt-get install libdb-dev
 
-    - name: Run instegration tests
+    - name: Run integration tests
       run: make test-integration
+
+  bench:
```

Comment:
We already have bench.yml
https://github.com/aquasecurity/fanal/blob/master/.github/workflows/bench.yml

Reply:
Ah yes – probably a rebase cruft. I'll remove this.